docs/user-manual/software/evaltool.md

```diff
@@ -49,22 +54,31 @@ The Info Bar can be seen from any tab and shows basic connection information for
 
 ## Update Firmware
 
-1. Enter the Settings tab.
-1. Open the COM ports of the units you would like to update.  If the units don't open up, you may have to change the baud rate.
-1. Click "Update Firmware".
-1. Choose the firmware file by clicking on the ellipsis (three dots) button next to the file name and navigating to the directory where your .hex file is located. Select it and press "Open".
-1. Repeat for the bootloader .bin file or check the bootloader update "Skip" checkbox. The bootloader will only be updated if the unit has an older bootloader than the file provided.
-1. Click "Start".
-1. Wait for the firmware to fully load or your units will enter into "bootloader mode" and you will have to reload the firmware again.
-1. After completion, click "Done".
+![](images/evaltool_firmware_update.png)
+
+1. Go to the **Settings** > **Serial Port** tab.
+1. Open the Ports of the units you would like to update.  If the units don't open up, you may have to change the baud rate.
+1. Click **Update Firmware**.
+1. Select the update type using the drop down menu:
+
+   | Update Type    | Description                                                  |
+   | -------------- | ------------------------------------------------------------ |
+   | FwPkg (.fpkg)  | Batch firmware update method for updating multiple devices in one process.  The .fpkg file contains multiple firmware files and instructions for sequencing firmware updates for all available devices. |
```

Comment:
Should probably mention IMX is not supported, yet.

Reply:
Might also be worth mentioning that this is the only mechanism to update GNSS receiver firmware.  Using GPX-1 below will not update GNSS receivers.